benches/misc.rs

```diff
@@ -62,6 +62,27 @@ fn misc_bernoulli_var(b: &mut Bencher) {
     })
 }
 
+macro_rules! sample_binomial {
+    ($name:ident, $n:expr, $p:expr) => {
+        #[bench]
+        fn $name(b: &mut Bencher) {
+            let mut rng = SmallRng::from_rng(&mut thread_rng()).unwrap();
+            let (mut n, mut p) = ($n, $p);
+            black_box((&mut n, &mut p));    // avoid constant folding
```

Comment:
This is the same problem the Bernoulli benchmarks have. `Binomial::new` can be lifted out of the loop by LLVM, so this line doesn't do much.

Reply:
You're right; removing this line has no impact. Interestingly moving the `Binomial::new` line up out of the loop makes misc_binomial_1 1ns faster (possibly) and misc_binomial_100 about 13ns slower.

Reply:
Keeping `Binomial::new` in the loop and using `black_box(&mut d)` on that has a more significant impact (makes the first two tests about 7ns slower).

Shall I just remove the `black_box` bit then?